fiaas_deploy_daemon/crd/watcher.py

```diff
@@ -72,8 +72,6 @@ def _watch(self, namespace):
     def _handle_watch_event(self, event: WatchEvent):
         if event.type in (WatchEvent.ADDED, WatchEvent.MODIFIED):
             self._deploy(event.object)
-        elif event.type == WatchEvent.DELETED:
-            self._delete(event.object)
```

Comment:
This doesn't seem right, the Kubernetes API will still generate these events and we shouldn't be raising any exception when receiving them.

Now that i check #101 I see it keeps the if check before raising.

Reply:
Good catch, you're right. Updating PR ...